package.json

```diff
@@ -3,6 +3,9 @@
   "version": "2.3.0",
   "type": "module",
   "packageManager": "pnpm@7.5.1",
+  "engines": {
+    "node": ">= 10.0.0"
```

Comment:
Let's remove this, since it's not only a node lib.

Reply:
Ok, but if it's not only a node lib, I will replace the `isAsyncFunction` with a dirty check implement because it maybe run in low version browser which is not support `async function` syntax.

Reply:
Let's not think about those old versions, I think what mitata does is good. But anyway, if you think your solution is better, feel free to.